lib/board.rb

```diff
@@ -4,6 +4,7 @@ class Board
   attr_reader :board
   attr_reader :POSITION_MIN
   attr_reader :POSITION_MAX
+  attr_accessor :counter
```

Comment:
Having a public accessor for something that is very internal to a board is something we should avoid.
What's stopping me as a client of board to call
```
board = Board.new
board.set_mark("X", 4)
board.counter = 42
```